src/content/docs/en/guides/client-side-scripts.mdx

```diff
@@ -46,7 +46,7 @@ In this example, adding the `<Hello />` component to a page will log a message t
 </script>
 ```
 
-### Script bundling
+### How Astro processes scripts
```

Comment:
```suggestion
### Script processing
```
Headings tend to be easier to scan when they are shorter, and "nouns" (things) people might care about. I think I'd have a preference for something like this?